File: Pascal/sumFunc.pas
program sumFunc;

var
  a, b, ret: integer;

function sum(num1, num2: integer): integer;

var
  result: integer;

begin
  result := num1+num2;
  sum := result;
end;

begin
  write('A: ');
  readln(a);
  write('B: ');
  readln(b);
  
  WriteLn(a, '+', b, '= ', sum(a,b));
  readKey();
end.

File: C/sumFunc.c
#include <stdio.h>
 //Compiler version gcc 6.3.0


int sum(int num1, int num2)
{
	int summ = num1 + num2;
	return summ;
}

int main(void)
{
	int a, b, res;
	printf("A: ");
	scanf("%d",a);
	
	printf("B: ");
	scanf("%d", b);
	
	res = sum(a, b);
	
	printf("%d",res);
}